dlc-manager/src/chain_monitor.rs

```diff
@@ -47,6 +47,10 @@ pub(crate) enum TxType {
     SplitTx,
     SettleTx,
     Cet,
+    // Introduced new variant to maintain backwards-compatibility.
+    SettleTx2 {
```

Comment:
❓ Do we need this backwards compatibility? There are no other instances where we force closed a settled dlc channel.

Reply:
I am not sure. I would still add it just in case. Eventually we should be able to confidently migrate all instances of `SettleTx` to `SettleTx2`, then drop `SettleTx` and rename `SettleTx2` to `SettleTx`.